filesmanager/filesmanager.py

```diff
@@ -304,6 +298,7 @@ def get_directories(self, data, suffix=''):
             }
         ]
         """
+        # self.prefill_directories()
```

Comment:
I'm commenting all `prefill_directories1 references in the meantime since we're not using the unpublished directory.